bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/product/VariantValueImpl.java

```diff
@@ -34,6 +38,24 @@ public void setLabel(String label) {
         this.label = label;
     }
 
+    @Override
+    public String getCssClassModifier() {
+        return defaultLabel;
+    }
+
+    public void setDefaultLabel(String defaultLabel) {
```

Comment:
java bean getter and setters should be named by the field `cssClassModifer`, `getCssClassModifier()`, `setCssClassModifier()`